src/debugger_requests.jl

```diff
@@ -352,11 +352,20 @@ function scopes_request(conn, state::DebuggerState, params::ScopesArguments)
 
     var_ref_id = length(state.varrefs)
 
+    scopes = []
+
     if isfile(file_name) && code_range !== nothing
-        return ScopesResponseArguments([Scope(name = "Local", variablesReference = var_ref_id, expensive = false, source = Source(name = basename(file_name), path = file_name), line = code_range.start, endLine = code_range.stop)])
+        push!(scopes, Scope(name = "Local", variablesReference = var_ref_id, expensive = false, source = Source(name = basename(file_name), path = file_name), line = code_range.start, endLine = code_range.stop))
     else
-        return ScopesResponseArguments([Scope(name = "Local", variablesReference = var_ref_id, expensive = false)])
+        push!(scopes, Scope(name = "Local", variablesReference = var_ref_id, expensive = false))
     end
+
+    curr_mod = JuliaInterpreter.moduleof(curr_fr)
```

Comment:
I forgot the detail, but does `moduleof` works for toplevel scope ?
Or we don't support that yet ?

Reply:
It does -- `moduleof ` is a no-op when given a module.

Reply:
aha, yes:

https://github.com/JuliaDebug/JuliaInterpreter.jl/blob/master/src/utils.jl#L6